apps/graphql/src/apps/booking/types/outputs/BookingInterface.js

```diff
@@ -66,7 +67,55 @@ export const commonFields = {
     type: GraphQLBookingType,
     description: 'OneWay, Multicity or Return',
   },
+  bagInfo: {
+    type: GraphQLList(GraphQLBag),
+    description: 'Total count of bags by the type',
+    resolve: ({ passengers }: Booking): Array<Bag> => {
+      const CABIN_BAG = 'Cabin bag';
+      const PERSONAL_ITEM = 'Personal item';
+      const CHECKED_BAGGAGE = 'Checked baggage';
+
+      const cabinBags = {
+        type: CABIN_BAG,
+        dimensions: '',
+        quantity: 0,
+      };
+      const personalItem = {
+        type: PERSONAL_ITEM,
+        dimensions: '',
+        quantity: 0,
+      };
+      const checkedBaggage = {
+        type: CHECKED_BAGGAGE,
+        dimensions: '',
+        quantity: 0,
+      };
+
+      passengers.forEach(passenger => {
+        passenger.bags.forEach(bag => {
+          switch (bag.type) {
+            case CABIN_BAG:
+              cabinBags.quantity += bag.quantity;
+              cabinBags.dimensions = bag.dimensions;
+              break;
+            case PERSONAL_ITEM:
+              personalItem.quantity += bag.quantity;
+              personalItem.dimensions = bag.dimensions;
+              break;
+            case CHECKED_BAGGAGE:
+              checkedBaggage.quantity += bag.quantity;
+              checkedBaggage.dimensions = bag.dimensions;
+              break;
+            default:
+              break;
+          }
+        });
+      });
+      return [personalItem, cabinBags, checkedBaggage];
```

Comment:
Please also write a test for this resolver you can use
`import { evaluateGraphQLResolver } from '@kiwicom/test-utils`

Reply:
Added bagInfo resolver test